common/types/status.go

```diff
@@ -4,18 +4,31 @@ package types
 // Status represents the stage that a swap is at.
 type Status byte
 
+// Status values
```

Comment:
Check these carefully as I didn't write the code they document. If we can get good comments on all the status and event types, we can close https://github.com/AthanorLabs/atomic-swap/issues/210.

Reply:
I think these are good!